pytorch_lightning/trainer/training_loop.py

```diff
@@ -347,7 +347,7 @@ def train(self):
                 if self.reload_dataloaders_every_epoch:
                     self.reset_train_dataloader(model)
                 # set seed for distributed sampler (enables shuffling for each epoch)
-                if (self.use_ddp or self.use_horovod) \
+                if (self.use_ddp or self.use_horovod or self.use_tpu) \
```

Comment:
I guess the variable here `on_tpu`

Reply:
the `use_tpu` variable is still used everywhere in the codebase. Just curious, when do you use `on_tpu` vs. `use_tpu`?

Reply:
I agree we shall clean this up...